test/integration/migrate/migration-integration-tests.js

```diff
@@ -649,6 +649,19 @@ module.exports = function (knex) {
               });
           });
       });
+
+      it('should drop a column with a default constraint (mssql)', function () {
+        return knex.migrate
```

Comment:
nit: you can use async-await here

Reply:
Oh, true. I just looked at how another test was done. Fixed.